plugins/postcss-minify/README.md

```diff
@@ -0,0 +1,100 @@
+# PostCSS Minify [<img src="https://postcss.github.io/postcss/logo.svg" alt="PostCSS Logo" width="90" height="90" align="right">][PostCSS]
+
+[<img alt="npm version" src="https://img.shields.io/npm/v/@csstools/postcss-minify.svg" height="20">][npm-url] [<img alt="Build Status" src="https://github.com/csstools/postcss-plugins/workflows/test/badge.svg" height="20">][cli-url] [<img alt="Discord" src="https://shields.io/badge/Discord-5865F2?logo=discord&logoColor=white">][discord]
+
+[PostCSS Minify] is a very basic CSS minifier.
+
+It guarantees two things :
+- browsers can not tell the difference between the original and the minified CSS
+- lawyers can not tell the difference between the original and the minified CSS
+
+It is not a CSS optimizer, it does not try to reduce the size of the CSS by altering the CSS itself.
+
+```pcss
+/*
+ * License : MIT-0
+ * legal text is preserved
+ */
+
+/*
+ * Copyright: CSSTools Authors
+ * legal text is preserved
+ */
+
+:root {
+	--mainColor: #12345678;
+}
+
+body {
+	color: var(--mainColor);
+	font-family: system-ui;
+}
+
+a {
+	color: rgb(0 0 100% / 90%);
+
+	&:hover {
+		color: rebeccapurple;
+	}
+
+	>span {
+		color: color-mix(in oklch, cyan, green 25%);
+	}
+}
+
+:is(input, button):is(:hover, :focus) {
+	color: oklch(40% 0.268735435 34.568626);
+}
+```
+
+becomes :
+
+```pcss
+/*
+ * License : MIT-0
+ * legal text is preserved
+ */
+
+/*
+ * Copyright: CSSTools Authors
+ * legal text is preserved
+ */ :root { --mainColor: #12345678 } body { color:var(--mainColor); font-family:system-ui } a { color:rgb(0 0 100% / 90%); &:hover { color:rebeccapurple } >span { color:color-mix(in oklch, cyan, green 25%) } } :is(input, button):is(:hover, :focus) { color:oklch(40% 0.268735435 34.568626) }
```

Comment:
I agree.

I've added paragraph in which I mention that it `only collapses whitespace and comments`. Hopefully this clarifies that it does not fully removes whitespace, even when it could be technically possible in some cases.